.env.sample

```
@@ -10,4 +10,6 @@ PORT= HTTP Server port
 COOKIE_SECRET=세션아이디
 EMAIL_USER=이메일보낼구글계정
 EMAIL_PASS=구글 앱비밀번호(발급필요)
-APP_NAME=메일보낼 앱이름
\ No newline at end of file
+APP_NAME=메일보낼 앱이름
+HOST=Redis 임의호스트이름
```

```suggestion
REDIS_HOST=Redis 임의호스트이름
```

HOST라는 이름은 혼동될 가능성이 커서 명시적으로 REDIS_ prefix를 붙이는게 좋을 것 같네요.